docs/runtime.md

```diff
@@ -66,6 +66,12 @@ decision] to replace rather than merge individual plugin configuration sections
 from imported configuration files. However, this behavior [may][containerd#7347]
 [change][containerd#9982] in future releases of containerd.
 
+Please note, that in order for drop-ins in `/etc/k0s/containerd.d` to take effect on running configuration, `/etc/k0s/containerd.toml` needs to be k0s managed. 
+
+If you change `/etc/k0s/containerd.toml` (by accident or on purpose), it automatically becomes "not k0s managed". To make it "k0s managed" again, remove it and restart k0s service on the node, it'll be recreated by k0s. 
```

Comment:
This statement doesn't sound correct:

https://github.com/k0sproject/k0s/blob/5fb4f1091ac0fda4017d7418e420ce13aeffa423/pkg/component/worker/containerd/component.go#L323-L328

Are you 100% certain that if the file is modified accidentally it's automatically not k0s managed in every case? If I understand correctly, this is only true if the first line is modified. Otherwise a restart of k0s should be enough to recreate the file.

Reply:
yes, you're right, only `# k0s_managed=true` matters, just tested it. 
I'll update the doc accordingly